asserts/snap_asserts.go

```diff
@@ -111,6 +112,21 @@ func assembleSnapDeclaration(assert assertionBase) (Assertion, error) {
 	}, nil
 }
 
+// SnapFileSHA3_384 computes the SHA3-384 digest of the given snap file.
+// It also returns its size.
+func SnapFileSHA3_384(snapPath string) (digest string, size uint64, err error) {
```

Comment:
yes, but this is inside the asserts package (and snap_asserts.go to be precise), it's used to compute the snap-sha3-384 field basically of the snap-revision/snap-build, I would keep it named like that until it's here and we really have no generic use case